app/gimlet/rev-b.toml

```diff
@@ -1280,10 +1282,17 @@ rx = { packets = 3, bytes = 1024 }
 [config.net.sockets.control_plane_agent]
 kind = "udp"
 owner = {name = "control_plane_agent", notification = "socket"}
-port = 11111 # TODO do we have a documented port for MGS traffic?
+port = 11111
 tx = { packets = 3, bytes = 2048 }
 rx = { packets = 3, bytes = 2048 }
 
+[config.net.sockets.dump_agent]
+kind = "udp"
+owner = {name = "dump_agent", notification = "socket"}
+port = 11113
```

Comment:
I know that this is somewhat unrelated, but worth (1) pointing to oxidecomputer/oana and/or (2) making oxidecomputer/oana public? (And I know that this might be a question for @rmustacc.)

Reply:
Good idea, added a comment to the top of the `[config.net]` section in all of the relevant TOML files